### ISSUE TITLE
Bug: Hint text not changing according to the language.

### ISSUE THREAD

**Issue by aashimawadhwa** [PalisadoesFoundation/talawa-admin#455]
**Describe the bug**
on changing the language the hint text in the Text filed remains in English and it does not look good.

**To Reproduce**

1. run talawa api and admin in your local machine
2. choose any other language expect English
3. see the issue on the signup page and login page

**Expected behavior**
The hint text should change as per the language.

**Actual behavior**
The hint text is not changing as of now.

**Screenshots**

![Screenshot 2023-02-09 at 2 05 34 PM](https://user-images.githubusercontent.com/73706697/217760597-a08f19d2-1d50-4a4d-b8d7-19dc4042496e.png)
![Screenshot 2023-02-09 at 2 05 14 PM](https://user-images.githubusercontent.com/73706697/217760633-c96e3d54-29c9-428e-a5e8-c6ffcb8e8a3a.png)
![Screenshot 2023-02-09 at 2 05 14 PM](https://user-images.githubusercontent.com/73706697/217760659-0f5aa63c-aca5-4274-b4ab-1db3f60a05b5.png)

**Comment by aashimawadhwa** [comment]
I would like to work on this issue.

**Comment by noman2002** [comment]
You can work on this and also check if this is happening anywhere else too.

**Comment by aashimawadhwa** [comment]
Yes it is happening on all the text input fields throughout the web application, I will fix them all in this PR.
